playbooks/common/openshift-cluster/upgrades/v3_5/storage_upgrade.yml

```diff
@@ -0,0 +1,19 @@
+---
+###############################################################################
+# Post upgrade - Upgrade job storage
+###############################################################################
+- name: Upgrade job storage
+  hosts: oo_first_master
+  roles:
+  - { role: openshift_cli }
+  vars:
+    # Another spot where we assume docker is running and do not want to accidentally trigger an unsafe
+    # restart.
+    skip_docker_role: True
+  tasks:
+  - name: Upgrade job storage
+    command: >
+      {{ openshift.common.client_binary }} adm --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+      migrate storage --include=jobs --confirm
+    run_once: true
```

Comment:
Would ops be running playbooks/byo/openshift-cluster/upgrades/v3_5/upgrade.yml for those upgrades or would they use .../v3_4/upgrade.yml ?